### DISCUSSION TITLE
FeatureOracleJavaSoft Does not seem to be implemented

### DISCUSSION THREAD

**Discussion by jax7778** [microsoft/openjdk#295] [Q&A] [answered]
Hello, 

I have been doing some testing with the MS Build of OpenJDK, and even though the installation page has the FeatureOracleJavaSoft" option listed, to overwrite the oracle Javasoft registry keys, the installers error out when you include that option. I am probably just missing something obvious here, but I just wanted to verify that this option is supported?

**Comment by d3r3kk** [comment, marked as answer]
When you say "the installers error out", what is the message you are seeing? Can you provide us with the logs?

**Reply by d3r3kk** [reply]
Potentially relevant: https://github.com/johanjanssen/TemurinChocolateyPackages/issues/15

**Reply by karianna** [reply]
@jax7778 Are you able to provide us the error log or details?

**Comment by apique13** [comment]
Hi, the FeatureOracleJavaSoft is missing in Microsoft OpenJDK installs and no JavaSoft is written (I checked it with an msi editor)
Still true in 17.0.7 and 11.0.19 downloaded from https://learn.microsoft.com/en-us/java/openjdk/download

**Reply by jax7778** [reply]
Yep,  that is what I was referring to. All you have to do is run the msi with that switch and it will fail

**Reply by brunoborges** [reply]
Hey folks, why is this important to you? What benefit this gives you today?

**Reply by jax7778** [reply]
Honestly it was more of a one off, I had an application that refused to run without the proper registry keys, I worked around it by using the adoptium package itself. It was not that crucial, so I didn't follow up on it, but I figured it should either be implemented, or removed from the documentation. For the few instances where it is helpful it might be nice.  I probably could have worked around it by manually adding in the keys it needed.

**Reply by GeoSimos** [reply]
It is quite important, especially in cases that we need to detect which type and version is installed on the system. For example, we do that via configuration baselines and PowerShell detection scripts via MCM (SCCM), in order to install certificates in Java's certificate store.

**Reply by d3r3kk** [reply]
@jmjaffe37 please keep this in mind while reworking our installer base this month!

**Comment by d3r3kk** [comment]
We will have a look, thanks for the report.

**Reply by karianna** [reply]
Issue #475
